src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -1023,7 +1023,9 @@ End Class
         <WorkItem(544299, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/544299")>
         <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
         Public Async Function TestExclusiveNamedParameterCompletion() As Task
-            Using state = TestState.CreateVisualBasicTestState(
+            Using state = TestState.CreateTestStateFromWorkspace(
```

Comment:
Why did this need to change?

Reply:
The intent of this change is to keep this test using a pre-VB15.5 language version.

The `CreateVisualBasicTestState` creates a workspace with a project using the default language version (which becomes VB16 after this PR, but was VB15 before). 
Since VB15 didn't have non-trailing named arguments, but VB15.5 (and thus VB 16) does, the test behavior changes.

Reply:
I don't think the intent here is for these two helpers to implicitly have different versions. That feels like a bug in itself. @dpoeschl?

Reply:
Resolved in offline discussion.

---
In reply to: [202852705](https://github.com/dotnet/roslyn/pull/28522#discussion_r202852705) [](ancestors = 202852705)